addons/medical/functions/fnc_addLoadPatientActions.sqf

```diff
@@ -0,0 +1,42 @@
+/*
+ * Author: 654wak654
+ * Adds child actions to the "load patient" action for near vehicles
+ *
+ * Arguments:
+ * 0: The medic <OBJECT>
+ * 1: The patient <OBJECT>
+ *
+ * Return Value:
+ * The child actions
```

Comment:
` <ARRAY>`

Also "The" is superfluous everywhere.

Reply:
I copied most things from other medical headers so I'm keeping all the "the"s.

Reply:
We should go for no "the"s, majority (I hope) is not using them, it's superfluous and quiet silly.

Reply:
at least for non-medical files